src/librbd/CMakeLists.txt

```diff
@@ -40,6 +40,7 @@ set(librbd_internal_srcs
   image/RefreshParentRequest.cc
   image/RefreshRequest.cc
   image/SetFlagsRequest.cc
+  image/RemoveRequest.cc
```

Comment:
Nit: try to preserve alphabetical ordering -- move between "RefreshRequest" and "SetFlagsRequest"